pkgs/development/python-modules/tempita/default.nix

```diff
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchFromGitHub, nose }:
+{ lib, buildPythonPackage, fetchurl, nose }:
 
 buildPythonPackage {
   version = "0.5.3-2016-09-28";
```

Comment:
It's "0.5.3" in the sources: https://bitbucket.org/ianb/tempita/src/97392d008cc8/setup.py